src/evaluator.ts

```diff
@@ -648,8 +648,9 @@ class Evaluator {
       for (let repeatIndex = 0; repeatIndex < numRepeat; repeatIndex++) {
         for (const vars of varCombinations) {
           let colIndex = 0;
-          for (const prompt of testSuite.prompts) {
-            for (const provider of testSuite.providers) {
+          // Order matters - keep provider in outer loop to reduce need to swap models during local inference.
```

Comment:
nit, put this note on the first occurrence (instead of or addition to the second)